src/Configurator/CopyFromPackageConfigurator.php

```diff
@@ -85,20 +111,51 @@ public function copyFile(string $source, string $target)
         if (file_exists($target)) {
             return;
         }
+
         copy($source, $target);
         @chmod($target, fileperms($target) | (fileperms($source) & 0111));
+        $this->write(sprintf(
+            'Created file <fg=green>"%s"</>',
+            $this->relativizePath($target)
+        ));
```

Comment:
Could you move this on one line? We rarely split a line (that is not important) on several lines.